src/DynamoCoreWpf/UI/GuidedTour/Step.cs

```diff
@@ -59,7 +96,8 @@ public enum PointerDirection { TOP_RIGHT, TOP_LEFT, BOTTOM_RIGHT, BOTTOM_LEFT };
         /// This property describe which will be the pointing direction of the tooltip (if is a Welcome or Survey popup then is not used)
         /// By default the tooltip pointer will be pointing to the left and will be located at the top
         /// </summary>
-        protected PointerDirection TooltipPointerDirection { get; set; } = PointerDirection.TOP_LEFT;
+        [JsonProperty("pointer_direction")]
```

Comment:
Curious if we have to introduce underscore and a different name in Json, can we unify them as much as possible?

Reply:
@QilongTang do you mean change it to  [JsonProperty("tooltip_pointer_direction")]? or removing the underscore?

Reply:
Sorry I was asking if we can use the same name

Reply:
This was fixed in the next comit: https://github.com/DynamoDS/Dynamo/pull/11895/commits/d41d7477b3079e8a54ab6fc642543f0017b4e422
Please consider that there are more cases like this in the json file and the Step class, please let me know if all the cases should be updated (like step_content, host_popup_info, etc...). Thanks

Reply:
All of them. Thanks!

Reply:
@QilongTang fixed all of them. Thanks
https://github.com/DynamoDS/Dynamo/pull/11895/commits/0b8ab72f225406baa13b1318ee6128df80dd46d8